metrics/validate_drp.yaml

```diff
@@ -0,0 +1,223 @@
+# validate_drp metric definitions
+# Definitions from LPM-17 (the Science Requirements Document).
+# http://ls.st/lpm-17
+
+PA1:
+  description: >
+    The maximum rms of the unresolved source magnitude distribution around the
+    mean value.
+  unit: mmag
+  reference:
+    doc: LPM-17
+    url: http://ls.st/lpm-17
+    page: 21
+
+PF1_minimum_gri:
+  description: >
+    The maximum fraction of magnitudes deviating by more than PA2_minimum_gri
+    from mean (%).
+  unit: ''
```

Comment:
Hmm. This is tricky, dimensionless_unscaled isn't a `str` identifier for a Unit:

```
>>> astropy.units.Unit('dimensionless_unscaled')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jsick/lsst/qa-system/lsstsw/miniconda/lib/python2.7/site-packages/astropy/units/core.py", line 1817, in __call__
    raise ValueError(msg)
ValueError: 'dimensionless_unscaled' did not parse as unit: At col 0, dimensionless_unscaled is not a valid unit.
```

But:

```
>>> astropy.units.Unit('') == astropy.units.dimensionless_unscaled
True
```

So... I think the empty string, `''` _is_ the official way of representing a dimensionless unscaled unit in astropy. We could add some custom logic to the `validate_base` parser layer that takes a string like `dimensionless_unscaled` and converts it into a `Unit`. Or we could try to PR this into astropy.